actors/evm/src/interpreter/precompiles/fvm.rs

```diff
@@ -133,9 +133,9 @@ pub(super) fn lookup_delegated_address<RT: Runtime>(
     Ok(ab)
 }
 
-/// Reads a FIL encoded address
-/// Resolves a FIL encoded address into an ID address
-/// returns BE encoded u64 or empty array if nothing found
+/// Reads a FIL encoded address.
+/// Resolves a FIL encoded address into an ID address.
+/// Returns BE encoded u256 (return will always be under 2^64). Empty array if nothing found or `InvalidInput` if length was larger 2^32.
```

Comment:
Since we mix up two forms of padding (right padding and left padding) in the precompiles, it wouldn't hurt to specify which kind of padding we use here (left/natural padding).

Reply:
will fix in follow up PR